src/components/PeopleTableRow/PeopleTableRow.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { findPersonByName } from '../../utils/findPersonByName';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+export const PeopleTableRow: React.FC<Props> = ({ person, people }) => {
+  const { personSlug } = useParams();
+  const { fatherName, motherName, sex, born, died, slug } = person;
+  const personFather = findPersonByName(fatherName, people);
+  const personMother = findPersonByName(motherName, people);
```

Comment:
Ensure that the `findPersonByName` function is robust enough to handle cases where the person is not found in the `people` array. It should return `null` or `undefined` appropriately to avoid errors when rendering `PersonLink` components.